src/fips/indicators.rs

```diff
@@ -841,6 +850,51 @@ const FIPS_CHECKS: FipsChecks = FipsChecks {
                 | CKF_UNWRAP
                 | CKF_DERIVE,
         },
+        FipsMechanism {
+            mechanism: CKM_TLS12_MASTER_KEY_DERIVE,
+            operations: CKF_DERIVE,
+            restrictions: [
+                restrict!(CKK_GENERIC_SECRET, step!(48 * 8)),
+                restrict!(),
+            ],
+            genflags: CKF_SIGN
+                | CKF_VERIFY
+                | CKF_ENCRYPT
+                | CKF_DECRYPT
+                | CKF_WRAP
+                | CKF_UNWRAP
+                | CKF_DERIVE,
+        },
+        FipsMechanism {
+            mechanism: CKM_TLS12_KEY_AND_MAC_DERIVE,
+            operations: CKF_DERIVE,
+            restrictions: [
+                restrict!(CKK_GENERIC_SECRET, range!(112, 255 * 64 * 8)),
+                restrict!(),
+            ],
+            genflags: CKF_SIGN
+                | CKF_VERIFY
+                | CKF_ENCRYPT
+                | CKF_DECRYPT
+                | CKF_WRAP
+                | CKF_UNWRAP
+                | CKF_DERIVE,
```

Comment:
Same Q as above

Reply:
the last one, `CKM_TLS12_KEY_SAFE_DERIVE`, should be treated the same way:

> CKM_TLS12_KEY_SAFE_DERIVE is identical to CKM_TLS12_KEY_AND_MAC_DERIVE except that it shall never produce IV data, and the  ulIvSizeInBits field of CK_TLS12_KEY_MAT_PARAMS is ignored and treated as 0.